tracpro/orgs_ext/tasks.py

```diff
@@ -14,6 +19,12 @@
 
 logger = get_task_logger(__name__)
 
+LAST_RUN_KEY = 'last_run_time:{task}:{org}'
+
+ORG_TASK_LOCK = 'org_task:{task}:{org}'
+
+LOCK_EXPIRE = 60  # 1 minute
```

Comment:
This needs to be long enough for the task to finish (not sure if it is) otherwise it can be acquired without being released.